src/components/file-viewer.jsx

```diff
@@ -26,12 +26,25 @@ class FileViewer extends Component {
   }
 
   componentDidMount() {
-    const container = document.getElementById('pg-viewer');
-    const height = container ? container.clientHeight : 0;
-    const width = container ? container.clientWidth : 0;
-    this.setState({ height, width });
+
+	  this.binded = this.onResize.bind(this)
```

Comment:
Let's keep the name of the event handler `onResize` instead of just `binded`. If we were to bind other methods in future this naming would become confusing.
```
this.onResize = this.onResize.bind(this);
```

Also, please make sure you use semicolons to stay consistent with our coding conventions.